torch/csrc/jit/codegen/cuda/iter_visitor.cpp

```diff
@@ -479,7 +479,7 @@ struct FindOutputs : public IterVisitor {
 
```

Comment:
This isn't about the changes in this PR, but is there any case where `out_stmt->isVal()` can be false?

Reply:
I don't think so, could likely be an assert.